src/test/kotlin/ru/iisuslik/ProtoTest.kt

```diff
@@ -0,0 +1,13 @@
+package ru.iisuslik
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class ProtoTest {
+    @Test
+    fun cosmasOuterClassHasBeenCompiled() {
+        assertEquals("CosmasOuterClass${'$'}VersionResponse",

```

Comment:
This test is nice, but apparently it tests Java naming conventions for nested classes :) The fact that protocol buffer has compiled and is available in the classpath is tested merely by accessing it in the second argument of `assertEquals`. 

If you just want a simple test, write `assertNotNull(CosmasOuterClass.VersionResponse.getDefaultInstance())`